projects/media-viewer/src/lib/toolbar/search-bar/search-bar.component.ts

```diff
@@ -81,15 +83,22 @@ export class SearchBarComponent implements OnInit, OnDestroy {
   }
 
   private setSearchResultsCount(results: SearchResultsCount): void {
-    this.haveResults = results.total > 0;
-    this.resultsText = this.haveResults
-      ? `${results.current} of ${results.total} matches`
-      : 'Phrase not found';
+    this.resultCount = results.total;
```

Comment:
is the resultCount variable needed, given we can just use results.total?

Reply:
resultCount is used by the HTML template to hide the next / prev links if there are no search results found